pkg/build/api/v1/types.go

```diff
@@ -371,6 +375,13 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty" description:"forces the source build to pull the image if true"`
 }
 
+// BuildPostCommitSpec holds a build post commit hook specification.
+type BuildPostCommitSpec struct {
+	// RunBash holds a bash script to be executed in a container running the
+	// build output image.
+	RunBash string `json:"runBash,omitempty" description:"bash script to be executed in a container running the build output image"`
```

Comment:
I think @smarterclayton will have last word on this, but I would prefer something amongs:

```javascript
"postCommit": {
  "command": "/any_command_that_acts as entrypoint",
  "args": "args to entrypoint"
}
```

That matches https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go#L850
The entrypoint for all S2I builders is /bin/bash afaik, which should work nicely.